packages/test/test-end-to-end-tests/src/test/stashedOps.spec.ts

```diff
@@ -128,6 +128,10 @@ describeCompat("stashed ops", "NoCompat", (getTestObjectProvider, apis) => {
 			configProvider: configProvider({
 				"Fluid.Container.enableOfflineLoad": true,
 			}),
+			options: {
+				// Smuggle in this option to pass down to DataStoreRuntime and IntervalCollection DDS
+				...({ intervalStickinessEnabled: true } as any),
```

Comment:
what did it do without this option?

Reply:
The test failed with an error saying it needed to be set

Reply:
ok. that is also wrong. @clarenceli-msft  i think this is related to some the stress issues you are looking at.